less/lists.less

```diff
@@ -111,7 +111,7 @@
     margin: auto;
     height: 100%;
     width: 100%;
-    font-size: 90%;
+    font-size: 0.84em;
```

Comment:
If the font size is set to `90%` elsewhere, then let's be homogenous by using the same value (in a perfect world, this would even be a less variable).
I suggest you also use a em-based font size for the search cards. Morover, I would simpify to 0.9em, rather than 0.84.